docs/topics/api/signing.rst

```diff
@@ -4,6 +4,19 @@ Signing
 
 .. note:: This API requires :doc:`authentication <auth>`.
 
+The following API endpoints help you get your add-on signed by Mozilla
+so it can be installed into Firefox without error. See
+`extension signing <https://wiki.mozilla.org/Addons/Extension_Signing>`_
+for more details about Firefox's signing policy.
+
+----------------
+Client Libraries
+----------------
+
+Instead of working with this API directly, you may wish to use the
+`jpm sign <https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpm#jpm_sign>`_
+command.
```

Comment:
I know its implied by the tool name but maybe say "If you are developing an add-on using the Add-on SDK and jpm, you may ..."